### PR TITLE
Add suchness

### PR DESCRIPTION
**Pull request by InbalAtTI**

### CODE REVIEW

**Comment by InbalAtTI** [comment]
Please review this, @casey-collab

File: english/unicorn/existence/existence.md [hunk 1]
```diff
@@ -2,4 +2,6 @@ The grey walls surrounding you look brighter than usual. The monotony continues.
 
 The porridge, at least, is [real](realporridge/porridge.md).
 
-This room, too, is an [illusion](fakeporridge/descartes.md).
\ No newline at end of file
+This room, too, is an [illusion](fakeporridge/descartes.md).
+
+Does it really [matter](justporridge/truth.md)?
```
Comment (casey-collab) [inline english/unicorn/existence/existence.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/unicorn/existence/justporridge/suchness.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Things are what they are and they are not.
```
Comment (casey-collab) [inline english/unicorn/existence/justporridge/suchness.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by InbalAtTI** [comment]
Please review again, think I fixed it, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :star:

**Pull request merged**